catalog/config-controller-monitoring/prometheus-instance.yaml

```diff
@@ -95,4 +80,53 @@ spec:
     targetPort: 9090
   selector:
     app: prometheus
-    prometheus: config-controller
\ No newline at end of file
+    prometheus: config-controller
+---
+# Promethus UI
+# https://cloud.google.com/stackdriver/docs/managed-prometheus/query#promui-deploy
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
note that deployment kinds aren't supported by Config Controller. So this would have to be added via the workaround. 

But we could merge these changes assuming that the Config Controller will help accommodate this use case.